builder-base/Dockerfile.ruby

```diff
@@ -1,126 +0,0 @@
-FROM ruby:2.5.1
```

Comment:
is generated [here](https://github.com/jenkins-x/jenkins-x-builders/pull/131/files#diff-e091688d80187842c1b91dcc19485fbaR104)